index.html

```diff
@@ -17,6 +17,6 @@
       </strong>
     </noscript>
     <div id="app"></div>
-    <!-- built files will be auto injected -->
+    <script type="module" src="/src/main.ts"></script>
```

Comment:
In Vite, the `index.html` is moved to the root directory and is changed accordingly. No more `<%= BASE_URL %>`.